postgres/init/22-portal-content.sql

```diff
@@ -420,6 +426,8 @@ ed4116b0-f4bc-451e-aaa9-5df08969e1b4	4ba33360-72e6-4a56-a253-59761d694d2c	\N	htt
 7f9888cb-2f78-4f71-b7dd-a9231888aa55	4d44eb62-7ac8-4a4a-a0d0-fc3daca5edd5	\N	\N	\N
 3d8cb906-76a3-4cd7-ae99-4f9225437a27	11fd3bb9-6922-4b36-bc28-d30aded20231	\N	https://dtc-translator.adac.openresearch.com	\N
 37241577-3730-4d39-ab5b-3555a0d11c42	1ab6c7c9-ff68-47da-aad4-b88d83b757cd	7e828901-27cb-4078-bd15-00efd247c56a	https://fleet-management.adac.openresearch.com	\N
+ac6413b0-b179-462a-9074-e7d0bcc5ee5a	1d28a224-6293-4148-8b60-6714a0f1465d	476b0600-965b-4f10-8eb5-e4568859a889	https://catena-x.palantirfoundry.com/	\N
```

Comment:
@Phil91 i think this is a duplicate now - I would delete the old row.  Do you agree?

Reply:
oh sorry, than i didn't look right while merging, thanks for double checking.
I agree can be deleted